camel-k-runtime-servlet/src/main/java/org/apache/camel/k/servlet/ServletRegistrationContextCustomizer.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.camel.k.servlet;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.component.servlet.CamelHttpTransportServlet;
+import org.apache.camel.k.ContextCustomizer;
+import org.apache.camel.k.Runtime;
+
+public class ServletRegistrationContextCustomizer implements ContextCustomizer {
+    public static final String DEFAULT_PATH = "/camel/*";
+
+    private String path;
+
+    public ServletRegistrationContextCustomizer(){
+        this(DEFAULT_PATH);
+    }
+
+    public ServletRegistrationContextCustomizer(String path){
+        this.path = path;
+    }
+
+    @Override
+    public void apply(CamelContext camelContext, Runtime.Registry registry) {
+        registry.bind(
+                "camel-servlet",
+                new ServletRegistration("CamelServlet", new CamelHttpTransportServlet(), path)
```

Comment:
we need ot be able to customize the name of the servlet, eventually we can use the name of the key used to bind the ServletRegistration to the camel registry

Reply:
done